meta/documents/user_guide/en/troubleshooting.md

```diff
@@ -0,0 +1,70 @@
+
+
+## Solving common issues
+
+Prior to contacting Wayfair, please use the table below to self-diagnose issues in the Wayfair plugin.
+
+| Issue | Possible causes |
+| ----- | --------------- |
+| General lack of data coming from Wayfair or going to Wayfair | <ul><li>Missing or incorrect credentials for the Wayfair Plugin in the active Plugin Set.</li><li>Internet connectivity issues</li><li>Temporary Wayfair downtime.</ul> |
+| Items missing from inventory sent to Wayfair | <ul><li>The `Item Mapping Method` setting in Wayfair's settings is set to the incorrect value.</li><li>The `Send all inventory items to Wayfair` setting is **disabled** and the Variation in Plentymarkets is not marked as available for the Wayfair marketplace.</li><li>The item is associated with a Plentymarkets Warehouse that is not mapped to a Wayfair Supplier ID on the `Warehouses` page of the Wayfair settings.</li><li>There is an empty value in the field of the Plentymarkets Variation that is associated with the `Item Mapping Method` setting in Wayfair's settings.</li><li>The amount of inventory for the item in Warehouses assigned to Wayfair Supplier IDs is 0, or the total amount is lower than the `Stock Buffer` setting.</li></ul>
+| No orders coming from Wayfair | <ul><li>The Wayfair Supplier ID for the Wayfair Purchase Order is not mapped to a Plentymarkets Warehouse on the `Warehouses` page of the Wayfair settings.</li></ul> |
+| Orders coming from Wayfair are not properly getting associated with Items in Plentymarkets | <ul><li>The `Item Mapping Method` setting in Wayfair's settings is set to the incorrect value.</li><li>There is an empty or incorrect value in the field of the Plentymarkets Variation that is associated with the `Item Mapping Method` setting in Wayfair's settings.</li></ul> |
+| Warehouses missing or incorrect for Order coming from Wayfair | <ul><li>Warehouses are not assigned to Wayfair Supplier IDs on the `Warehouses` page of the Wayfair settings.</li><li>Multiple Warehouses associated with one Wayfair Supplier ID on the `Warehouses` page of the Wayfair settings.</li></ul> |
+| Shipping label download button does not appear in the "Overview" tab of an Order in the Plentymarkets Shipping centre | <ul><li>Order's shipment has not been registered.</li><li>Shipping Centre was not yet refreshed after shipment registration was performed.</li><li>Document downloading failed during the shipment registration process.</li></ul> |
+
+## Reporting issues to Wayfair
+
+If you are unable to resolve an issue with the Wayfair plugin, you may file a report on [the issues page for the plugin](https://github.com/wayfair-contribs/plentymarkets-plugin/issues) so that it may be evaluated by the Wayfair team.
+
+**When contacting Wayfair, please make sure to include information on any settings that have recently been changed in Plentymarkets or in the Wayfair plugin's settings, and be prepared to answer questions about your specific configuration.**

```

Comment:
**Rephrase to:**
When contacting Wayfair, please include information on any settings that have recently been changed in Plentymarkets or in the Wayfair plugin's settings. To provide the best support, we may also need to ask you additional questions about your specific configuration.